clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/HelixParticipationService.java

```diff
@@ -360,6 +369,28 @@ private void asyncStart() {
     });
   }
 
+  static void resetAllInstanceCVStates(
+      HelixPartitionStatusAccessor accessor,
+      StorageService storageService,
+      Logger currentLogger) {
+    // Get all hosted stores
+    currentLogger.info("Started resetting all instance CV states");
+    Map<String, Set<Integer>> storePartitionMapping = storageService.getStoreAndUserPartitionsMapping();
+    storePartitionMapping.forEach((storeName, partitionIds) -> {
+      currentLogger.info("Started deleting replica status for resource: {}, partitions: {}", storeName, partitionIds);
```

Comment:
I think we can get rid of this... no need for logs before and after. We are a bit sensitive to excessive logging these days so it's better not to pile on too much more, IMHO. In a cluster with 500 stores this line will come up a lot during startup and shutdown...

Reply:
I added this logging intentionally to monitor the performance of CV state cleanup since it will cleanup a CV state per partition.

If you think per-store logging is too much, I can only keep the log for the beginning and end.

Reply:
If you want the per-store timing you could take the time before and after and include that in the log?